portfolio/src/main/java/com/google/sps/servlets/metMylesServlet.java

```diff
@@ -35,16 +35,23 @@
 @WebServlet("/met-Myles")
 public class metMylesServlet extends HttpServlet {
 
-  public Map<String, Integer> metVotes = new HashMap<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Votes").addSort("timestamp", SortDirection.DESCENDING);
 
-    //Makes a new datastore object and intializes it with the comment
+    //Makes a new datastore object and intializes it with the vote
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-    
+
+    Map<String, Integer> metVotes = new HashMap<>();
+
+    for (Entity entity: results.asIterable()) {
+        String key = (String) entity.getProperty("title");
+        int currentVotes = metVotes.containsKey(key) ? metVotes.get(key) : 0;
+        metVotes.put(key, currentVotes + 1);
+    }
+
     //prints out to chart container
```

Comment:
this doesn't really print, so much as convert to a json response

Reply:
good point, i'll change it